client/package.json

```diff
@@ -8,6 +8,8 @@
     "@testing-library/user-event": "^13.5.0",
     "chart.js": "^4.2.1",
     "d3": "^7.8.2",
+    "express-async-handler": "^1.2.0",
+    "mongoose": "^7.0.1",
```

Comment:
Could you kindly help me understand why `express-async-handler` and `mongoose` have been added to the frontend? They are typically used in backend.